xx.md

```diff
@@ -0,0 +1,46 @@
+NIP-XX
+======
+
+Audio Events
+---------------
+
+`draft` `optional`
+
+This NIP defines a new event kind `31337` representing an audio track. These events are _parameterized replaceable_
+and deletable per [NIP-09](09.md).
+
+Unlike a `kind 1` event with audio attached, audio events are intended to be surfaced in an audio-specific context
+rather than in a general micro-blogging context.
+
+A kind `31337` event's `content` should be summary or description of the audio content.
+
+The following tags are required:
+
+- `d` is a unique identifier. This MAY be generated by the client creating the event, or refer to an external GUID.
+- `media` is the url of the media. This SHOULD NOT be read, but is required for backwards compatibility reasons.
+- `imeta` is as described in NIP 92. Multiple `imeta` tags MAY be included to provide for different use cases
+  (e.g. streaming vs download).
+- `subject` is the title of the audio track
+
+The following tags are optional:
+
+- `website` is an external url to a website related to the track.
+- `creator` is the pubkey (or plain text name) of the track author(s).
```

Comment:
I think this should be indexable, probably a `p` tag and if it's not a pubkey `n` or something else; the use case of "more songs by this artist" is very normal

Reply:
please use "something else" instead of `n` cause the tag is being used to add events to unbounded lists - set to the name of the list - like in [this example](https://github.com/nostr-protocol/nips/pull/761/files#diff-55a569d18c00c9db28ae64db89efdfd5cb644b26d2a9333df750e3ca385e5452R41-R42).

In theory, any event kind, specially replaceable ones, would be able to use the `n` tag for adding events to these lists.

Reply:
I've updated this to use creator + p tag (creator so a mark can be added, p tag so that we have a standard format). This is slightly incompatible with zapstr's current p tags, but not enough to break anything (zapstr's role will be interpreted as a relay hint).

Reply:
sorry about the missing relay hint

is the idea then that each creator will be tagged twice? once as a `creator` and once as a `p`?

Reply:
Right, so you can query by `p`, but you can still tag people who weren't creators using `p` tags. Without the `creator` tag the semantics of `p` becomes overloaded, so it's necessary to denote the person's role in contributing to the track (and support plaintext creator names, which wouldn't have an accompanying p tag).